setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
     social-auth-core[openidconnect]==3.2.0
     social-auth-app-django==3.1.0
     django-redis>=4.11.0
-    django-treebeard==4.4
+    django-treebeard
```

Comment:
As you unpin, `django-treebeard` and `Django`, you should remove these dependencies to the list of python dependencies ignored by renovate (`/renovate.json`)

Reply:
<img src="https://user-images.githubusercontent.com/1427165/128152423-70de0d0d-3116-4e10-afa9-275071539a1f.jpg" width="100">